test/functional/manifest.ini

```diff
@@ -1,5 +1,7 @@
 [DEFAULT]
-run-if = buildapp == 'browser'
+# app == 'Firefox' is used for running in the Loop repo.
+# buildapp == 'browser' is used for mozilla-central & co.
+run-if = buildapp == 'browser' || app == 'Firefox'
```

Comment:
I'm confused about the buildapp part.  That looks like it would mean that the functional test would be run by default in mozilla-central Mn tests, which, as far as I know, is not what's happening.  Can you explain, and if you think it's worth it, add a comment?